File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for(int i = 1; i <= n; i++){ // 약수 1부터 n까지 나누기 반복
            int cnt = 0;
            for(int j = 1; j <= n; j++){ // 1부터 n까지 나눌 수
                if(i % j == 0)  cnt++;
            }
            if(cnt >= 3)    answer++;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    answer = 0
    
    for num in range(1, n + 1):
        cnt = 0
        for div in range(1, num + 1):
            if num % div == 0:
                cnt += 1
                
            if cnt >= 3:
                answer += 1
                break
    return answer